cranelift-codegen/src/ir/mod.rs

```diff
@@ -74,6 +74,7 @@ pub type SourceLocs = SecondaryMap<Inst, SourceLoc>;
 
 /// Marked with a label value.
 #[derive(Copy, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "enable-serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
My personal taste would be avoiding the `serde::` by adding the extra `use` declaration. I'm not sure about our code conventions though.